portal/src/graphql/portal/PasswordPolicySettings.tsx

```diff
@@ -63,7 +63,7 @@ function constructStateFromAppConfig(
   const historySize = passwordPolicy?.history_size ?? 0;
 
   return {
-    minLength: passwordPolicy?.min_length ?? 0,
+    minLength: passwordPolicy?.min_length ?? NaN,
```

Comment:
I usually store numeric form fields as string. And only convert them to number on submission. Would this approach be simpler? For this particular form, `minLength`, `historyDays` and `historySize` should be stored as string.

Reply:
> I usually store numeric form fields as string. And only convert them to number on submission. Would this approach be simpler?

I agree, this is what I usually do as well. I keep it as number as the least changes has to be made. Can change it to string. Using NaN as empty is not a good idea anyway